pkg/minikube/cluster/cluster.go

```diff
@@ -344,9 +347,21 @@ To disable this message, run [minikube config set ShowDriverDeprecationNotificat
 	}
 }
 
+func megs(bytes uint64) int {
+	return int(bytes / 1024 / 1024)
+}
+
 func createHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error) {
 	preCreateHost(&config)
-	console.OutStyle(console.StartingVM, "Creating %s VM (CPUs=%d, Memory=%dMB, Disk=%dMB) ...", config.VMDriver, config.CPUs, config.Memory, config.DiskSize)
+	if config.VMDriver != constants.DriverNone {
```

Comment:
just a nit, but maybe put this logic into a new helper function ? showHostInfo() ?

Reply:
Sounds like a good idea!